keel-clouddriver/keel-clouddriver.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
   implementation("org.springframework:spring-context")
   implementation("org.springframework.boot:spring-boot-autoconfigure")
   implementation("com.netflix.frigga:frigga")
+  implementation("com.netflix.spectator:spectator-reg-micrometer")
```

Comment:
This already contains a metrics implementation for caffeine caches

Reply:
Nice...

Reply:
@cfieber pointed me to it